README.md

```diff
@@ -38,3 +38,33 @@ A processor to retrieve metadata about a CKAN resource from a CKAN instance and
 - `ckan-host`: The base url (and scheme) for the CKAN instance (e.g. http://demo.ckan.org).
 - `resource-id`: The id of CKAN resource
 - `ckan-api-key`: Either a CKAN user api key or, if in the format `env:CKAN_API_KEY_NAME`, an env var that defines an api key. Optional, but necessary for private datasets.
+
+### `ckan.dump.to_ckan`
+
+A processor to save a datapackage and resources to a specified CKAN instance.
+
+```yaml
+  run: ckan.dump.to_ckan
+  parameters:
+    ckan-host: http://demo.ckan.org
+    ckan-api-key: env:CKAN_API_KEY
+    overwrite_existing: true
+    dataset-properties:
+      name: test-dataset-010203
+      state: draft
+      private: true
+      owner_org: my-test-org
+```
+
+- `ckan-host`: The base url (and scheme) for the CKAN instance (e.g. http://demo.ckan.org).
+- `ckan-api-key`: Either a CKAN user api key or, if in the format `env:CKAN_API_KEY_NAME`, an env var that defines an api key.
+- `overwrite_existing`: If `true`, if the CKAN dataset already exists, it will be overwritten by the datapackage. Optional, and default is `false`.
+- `dataset-properties`: An optional object, the properties of which will be used to set properties of the CKAN dataset.
+
+##### CKAN dataset from datapackage
+
+The processor first creates a CKAN dataset from the datapackage specification, using the CKAN api [`package_create`](http://docs.ckan.org/en/ckan-2.6.4/api/#ckan.logic.action.create.package_create). If the dataset already exists, and parameter `overwrite_existing` is `True`, the processor will attempt to update the CKAN dataset using [`package_update`](http://docs.ckan.org/en/ckan-2.6.4/api/#ckan.logic.action.update.package_update). All existing resources and dataset properties will be overwritten.
```

Comment:
Can you use /ckan-2.7.2/ or even /latest/?

Reply:
Slip of a google search.